pyproject.toml

```diff
@@ -84,13 +84,13 @@ cz_jira = "commitizen.cz.jira:JiraSmartCz"
 cz_customize = "commitizen.cz.customize:CustomizeCommitsCz"
 
 [tool.poetry.plugins."commitizen.provider"]
-cargo = "commitizen.providers:CargoProvider"
-commitizen = "commitizen.providers:CommitizenProvider"
-composer = "commitizen.providers:ComposerProvider"
-npm = "commitizen.providers:NpmProvider"
-pep621 = "commitizen.providers:Pep621Provider"
-poetry = "commitizen.providers:PoetryProvider"
-scm = "commitizen.providers:ScmProvider"
+cargo = "commitizen.providers.cargo_provider:CargoProvider"
```

Comment:
Should we add a `commitizen/providers/__init__.py` to avoid this change?

Reply:
I originally added them all to `__init__.py`, but ruff removed them when I ran `scripts/format` due to  F401 (unused import). I can override in `__init__.py`, if that makes sense for you.

Reply:
@Lee-W - went ahead and added `# noqa: F401` to the import lines in `__init__.py` and reverted the changes in `pyproject.toml`. Let me know if you want this handled differently.

Reply:
Sounds good